Neos.Media/Classes/Domain/Model/Dto/AssetResourceReplaced.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Neos\Media\Domain\Model\Dto;
+
+use Neos\Flow\ResourceManagement\PersistentResource;
+use Neos\Media\Domain\Model\AssetInterface;
+
+/**
+ *
```

Comment:
It seems there is a little place of a doc comment already anticipated and i would very much like this.